meta/st0x-spec.md

```diff
@@ -0,0 +1,87 @@
+# st0x Server API Specification V1
+
+## Goal
+
+Provide signed price quotes for specific trading pairs that can be used by solvers in Raindex orders. Each endpoint handles a single trading pair and returns a signed context containing quote information.
+
+## Magic Numbers
+
+- Web data metadata magic number: `0xff5dcce9b571ba42`
+- st0x API magic number: `0x7374307870616972` ("st0xpair" in hex)
```

Comment:
the point of a magic number is for it to have a negligible chance of colliding with any file that isn't what we're expecting, so that we don't misinterpret data

so at the least it should start with `0xff` but it also shouldn't be a valid ascii string that could easily be found in the wild in other contexts

safer would be to use the initial bytes of the hash of `st0xpair` rather than literally the string, which would likely result in at least a few invalid ascii/utf8 chars and prevent it matching random strings

Reply:
ah ok - i'll make another one